### ISSUE TITLE
WinUI 3 Sample

### ISSUE THREAD

**Issue by michael-hawker** [CommunityToolkit/MVVM-Samples#5]

**Comment by LuohuaRain** [comment]
Is there any progress on WinUI3 project reunion 0.8 preview? Thanks.

**Comment by suchja** [comment]
@michael-hawker Are there any plans to provide a dedicated sample for WinUI3?

**Comment by michael-hawker** [comment]
@suchja yes we plan to add more samples, just haven't gotten to it yet. Usage is the same as the UWP one, just with the `CommunityToolkit.Mvvm` package which we're aligning on for all platforms for our next version.
